go/oasis-node/cmd/common/common.go

```diff
@@ -278,3 +278,17 @@ func LoadEntity(signerBackend string, entityDir string) (*entity.Entity, signatu
 
 	return entity.Load(entityDir, factory)
 }
+
+func ExtractEntity(signerBackend string, entityDir string) error {
```

Comment:
Add documentation comment, consider renaming (also below you call this exporting) as this just creates an empty entity from a public key?